homeassistant/scripts/keyring.py

```diff
@@ -0,0 +1,44 @@
+"""Script to get, set and delete secrets stored in the keyring."""
+import argparse
+import getpass
+
+from homeassistant.util.yaml import _SECRET_NAMESPACE
+
+REQUIREMENTS = ['keyring']
```

Comment:
Please specify a version.

Also, the handling of REQUIREMENTS is not part of this PR.